packages/components/flex/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex as ChakraFlex, ThemeProvider } from '@chakra-ui/core';
+
+import theme from '../../helpers/theme';
+
+const Flex = ({ children, ...props }) => (
+  <ThemeProvider theme={theme}>
```

Comment:
Do we need to pass theme inside the component ?

Reply:
It's actually asked by @navin-moorthy  to pass the theme inside the component.

Reply:
Need to find a workaround for this.

Reply:
Yeah. I will find a way. HOC is not working with dot notation components. @prasanna1211 when you have time.